modules/webrtc/doc_classes/WebRTCDataChannel.xml

```diff
@@ -11,85 +11,106 @@
 			<return type="void">
 			</return>
 			<description>
+				Closes this data channel, notifying the other peer.
 			</description>
 		</method>
 		<method name="get_id" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the id assigned to this channel during creation (or auto-assigned during negotiation).
+				If the channel is not negotiated out-of-band the id will only be available after the connection is established (will return [code]65535[/code] until then).
 			</description>
 		</method>
 		<method name="get_label" qualifiers="const">
 			<return type="String">
 			</return>
 			<description>
+				Returns the label assigned to this channel during creation.
 			</description>
 		</method>
 		<method name="get_max_packet_life_time" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the maxPacketLifeTime value assigned to this channel during creation.
+				Will be [code]65535[/code] if not specified.
 			</description>
 		</method>
 		<method name="get_max_retransmits" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the maxRetrismits value assigned to this channel during creation.
+				Will be [code]65535[/code] if not specified.
 			</description>
 		</method>
 		<method name="get_protocol" qualifiers="const">
 			<return type="String">
 			</return>
 			<description>
+				Returns the sub-proctocol assigned to this channel during creation. An empty string if not specified.
 			</description>
 		</method>
 		<method name="get_ready_state" qualifiers="const">
 			<return type="int" enum="WebRTCDataChannel.ChannelState">
 			</return>
 			<description>
+				Returns the current state of this channel, see [enum WebRTCDataChannel.ChannelState].
 			</description>
 		</method>
 		<method name="is_negotiated" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if this channel was created with out-of-band configuration.
 			</description>
 		</method>
 		<method name="is_ordered" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if this channel was created with ordering enabled (default).
 			</description>
 		</method>
 		<method name="poll">
 			<return type="int" enum="Error">
 			</return>
 			<description>
+				Reserved, but not used for now.
```

Comment:
Side note for @godotengine/documentation : how do we manage documentation for reserved methods?